src/internal/transactionSubmission.ts

```diff
@@ -347,11 +374,11 @@ export async function rotateAuthKey(args: {
         MoveVector.U8(proofSignedByNewPrivateKey.toUint8Array()),
       ],
     },
+    abi: rotateAuthKeyAbi,
   });
-  const pendingTxn = await signAndSubmitTransaction({
+  return signAndSubmitTransaction({
```

Comment:
I think we can make a eslint rule for that

Reply:
We can, will probably do a pass over it later, but it seemed to be very common.